core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1793,6 +1793,22 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.ARRAY_OF_INTEGER,
           SqlFunctionCategory.STRING);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
```

Comment:
Nit: would you mind adding javadocs here? I know not all of the functions have them but given how many functions we've been adding to Calcite recently I think it would be a good practice to have them well-commented.

Reply:
Add some comments for three `code_points` functions.